src/dk/aau/cs/ds306e18/tournament/rlbot/RLBotSettings.java

```diff
@@ -24,16 +28,25 @@ public void setMatchConfig(MatchConfig matchConfig) {
         this.matchConfig = matchConfig;
     }
 
+    public boolean writeOverlayData() {
```

Comment:
```suggestion
    public boolean writeOverlayDataEnabled() {
```

Makes it clearer what the function of the function is.